spec/factories/samples.rb

```diff
@@ -46,6 +46,8 @@
         create(:metadatum, key: key, raw_value: value, sample: sample, metadata_field: MetadataField.find_by(name: key))
       end
 
+      # Pipline runs create pipeline_run_stages automatically, which require a human host genome record
+      HostGenome.find_by(name: "Human") || create(:host_genome, name: "Human")
```

Comment:
THis should be in the pipeline run factory, no?

Reply:
Oops — didn't realize that FactoryBot has a before create hook. Will update!